frame/staking/src/tests.rs

```diff
@@ -3006,7 +3060,7 @@ mod offchain_phragmen {
 					winners,
 					compact,
 					score,
-					active_era(),
+					current_era(),
```

Comment:
note that for all those test current_era == active_era anyway.

Reply:
What would be substantially better is to change the test setup to also have this one session drift. This is probably a lot of work, but I think a good step forward. In that case, I would expect active_era and current_era difference to matter. 

Also related.

Reply:
indeed if we want that we need to use a different session-pallet which have more than one session delay.